README.rst

```diff
@@ -46,11 +75,51 @@ At the command line, run the command::
 
 This will install this module and all required dependencies.
```

Comment:
Does this mean it will also install mbstring and curl? Or what dependencies are these?